ProcessLib/SmallDeformation/CreateSmallDeformationProcess.cpp

```diff
@@ -98,8 +98,26 @@ createSmallDeformationProcess(
         config.getConfigParameter<double>(
             "reference_temperature", std::numeric_limits<double>::quiet_NaN());
 
+    const auto material_ids =  materialIDs(mesh);
+
+    std::vector<int> deactivated_subdomains;
+    if (material_ids)
+    {
+        auto deactivated_subdomains_optional =
+            //! \ogs_file_param{prj__processes__process__SMALL_DEFORMATION__deactivated_subdomains}
+            config.getConfigParameterOptional<std::vector<int>>(
+                "deactivated_subdomains");
+        if (deactivated_subdomains_optional)
+        {
+            std::copy(deactivated_subdomains_optional->begin(),
+                      deactivated_subdomains_optional->end(),
+                      std::back_inserter(deactivated_subdomains));
```

Comment:
Why not simply `deactivated_subdomains = std::move(*deactivated_subdomains_optional)`?

Reply:
or `auto deactivated_subdomains = config.getConfigParameter(
                "deactivated_subdomains", std::vector<int>{});`

Reply:
Changed.